packages/sdk/src/updateRole.test.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @group with-v2-entitlements
+ */
+
+import { createTownWithRequirements, updateRole } from './util.test'
+import { NoopRuleData, UpdateRoleParams, Permission, EVERYONE_ADDRESS } from '@river-build/web3'
+
+describe('updateRole', () => {
+    test('user-gated space created with no-op ruleData allows updates on minter role', async () => {
+        const { bobSpaceDapp, bobProvider, spaceId } = await createTownWithRequirements({
+            everyone: false,
+            users: ['alice'],
```

Comment:
should this say 'bob' instead of 'alice'?

Reply:
no, literally doesn't matter, just didn't want it to be empty.